app/controllers/merchant/items_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Merchant::ItemsController < ApplicationController
+  before_action :require_merchant
+
+  def index
+    @items = current_user.merchants[0].items
+  end
```

Comment:
at this point have access to current_user, so passing items to the items index page through the current user and their merchant connection

Reply:
I am wondering why we need to index merchants here. Shouldn't a current user only ever have one merchant connection?

Reply:
yeah i'm not sure what the index position is about, does current_user.merchants return an array type active record association? that might be why

Reply:
yes, it's a many to many that returns an array of merchants